java-tools/OsmAndServer/src/main/java/net/osmand/server/api/services/PromoService.java

```diff
@@ -40,7 +42,7 @@ public void register(PromoCampaignRepository.Promo promo) {
 	}
 
 	@Transactional
-	public ResponseEntity<String> addUser(String name, String email) {
+	public ResponseEntity<String> addUser(String name, String email) throws FileNotFoundException, UnsupportedEncodingException {
```

Comment:
Exceptions shuold be RuntimeException by guidelines as we don't need here checked exceptions

Reply:
now caught locally